### ISSUE TITLE
Add preview thumbnail composables

### ISSUE THREAD

**Issue by padmalcom** [Chaintech-Network/ComposeMultiplatformMediaPlayer#38]
Hi, this is more an enhancement request. I display a lot of videos in a list. It would be nice if those videos could be loaded in a thumbnail composable instead of loading the entire videos.

**Comment by ctn-mobile-innovation** [comment]
@padmalcom Good suggestion! to be more specific can you give us example of same? we will add in next version.

**Comment by padmalcom** [comment]
Yes sure, for all supported video formats it would be nice to display a single frame as Image like composable as a kind of placeholder to give the user an idea of the video content.

You could enhace that by a composable that displays the e.g. 5 most significant frames in a loop to have a moving preview of a video.

**Comment by ctn-mobile-innovation** [comment]
Please check latest version for this!

**Comment by padmalcom** [comment]
@ctn-mobile-innovation: Awesome job, thank you!
